src/docfx/serve/LanguageServerBuilder.cs

```diff
@@ -73,16 +75,20 @@ public async void Run(CancellationToken cancellationToken = default)
                     // The current build can only be completed when the credential refresh request response get handled.
                     // But the responses of language server are handled sequentially, which cause the deadlock.
                     await Task.Yield();
+                    Telemetry.SetIsRealTimeBuild();
```

Comment:
net to `SetIsReadTimeBuild(false)` in a `finally` block, otherwise the value will be leaked to non real-time builds.